src/libaktualizr/primary/sotauptaneclient.cc

```diff
@@ -425,16 +423,16 @@ bool SotaUptaneClient::updateImagesMeta() {
       local_version = -1;
     }
 
-    if (local_version > remote_version) {
-      return false;
-    } else if (local_version == remote_version) {
-      return true;
-    }
     if (!images_repo.verifyTargets(images_targets)) {
       last_exception = images_repo.getLastException();
       return false;
     }
-    storage->storeNonRoot(images_targets, Uptane::RepositoryType::Images, Uptane::Role::Targets());
+
+    if (local_version > remote_version) {
+      return false;
+    } else if (local_version == remote_version) {
```

Comment:
Why is this case an `==` but the others are all `<`?

Reply:
That's an error, yes, thanks.